.github/workflows/roborazzi_tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Roborazzi Integration Tests
+
+on:
+  push:
+    branches: [ master, 'robolectric-*.x' ]
+    paths-ignore:
+      - '**.md'
+
+  pull_request:
+    branches: [ master, google ]
```

Comment:
We can run roborazzi tests on the Google branch, that is fine with me.

Importing a third party library into Google is a lengthy process so I don't think it will be supported in the near future.